examples/frameworks/auto_wrapper/langchain_deep_research/langgraph_deep_research.ipynb

```
@@ -0,0 +1,1246 @@
+    "# Integrating Existing LangGraph Agents with NeMo Agent Toolkit\n",
+    "\n",
+    "In this notebook, you'll learn how to integrate any existing LangGraph agent with NeMo Agent toolkit using the `langgraph_wrapper` workflow type.\n",
+    "\n",
+    "We'll use LangGraph's Deep Research agent as a comprehensive example to demonstrate how you can wrap existing LangGraph agents so they work seamlessly with NeMo Agent toolkit features like configurable LLMs, telemetry and observability with Phoenix, and comprehensive evaluation frameworks-all without refactoring the original agent code.\n",
+    "\n",
+    "The techniques shown here apply to any LangGraph agent, making it easy to add NeMo Agent toolkit's powerful capabilities to your existing LangGraph applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)